lib/markdown_generators.js

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env node
+
+const yaml = require("js-yaml");
+const markdown = require("markdown-builder");
+const title_case = require("title-case");
+const TurndownService = require("turndown");
+const tables = require("turndown-plugin-gfm").tables;
+const turndown_service = new TurndownService();
+turndown_service.use(tables);
+const helpers = require("./helpers");
+
+class markdown_generators {
```

Comment:
This is redundant for the file. We can export the functions below directly instead, for example

    module.exports = {
      generate_course_home_front_matter,
      generate_course_section_front_matter
      ...
    }